scipy/interpolate/_rgi.py

```diff
@@ -278,6 +278,12 @@ def __call__(self, xi, method=None):
         values_x : ndarray, shape xi.shape[:-1] + values.shape[ndim:]
             Interpolated values at `xi`.
 
+        Notes
+        -----
+        In the case that ``xi.ndims == 1`` a new axis is inserted into
```

Comment:
Typo: `x.ndim`. Also maybe spell out explicitly that it's about the `values_x` array (the same name is in the Returns section). And maybe add "See Notes" or some such to the return value description in the Returns section.